src/xrCore/_std_extensions.cpp

```diff
@@ -15,7 +15,12 @@ char* timestamp(string64& dest)
 {
     time_t     now = time(nullptr);
     struct tm  tstruct;
-    tstruct = *localtime(&now); // localtime is NOT thread-safe
+
+#if defined(WINDOWS)
+    localtime_s(&tstruct, &now);// thread-safe for windows
```

Comment:
According to documentation, it's recommended to define `__STDC_WANT_LIB_EXT1__` in `Common/PlatformWindows.inl`
https://en.cppreference.com/w/c/chrono/localtime

Reply:
Ah, we don't need it for Windows.